code/modules/mob/living/carbon/human/say.dm

```diff
@@ -5,6 +5,13 @@
 		verb_ask = pick(T.ask_mod)
 		verb_yell = pick(T.yell_mod)
 		verb_exclaim = pick(T.exclaim_mod)
+	if(wear_mask)
+		var/obj/item/clothing/mask/M = wear_mask
+		if(!isnull(M.chosen_tongue))
+			verb_say = pick(M.chosen_tongue.say_mod)
+			verb_ask = pick(M.chosen_tongue.ask_mod)
+			verb_yell = pick(M.chosen_tongue.yell_mod)
+			verb_exclaim = pick(M.chosen_tongue.exclaim_mod)
```

Comment:
Rather than create a new tongue inside the mask, which could be a little odd, store the typepath of the tongue that you want to use and use initial() in order to determine which mods to use.
```
var/datum/example/example_typepath = /datum/example/tester
verb_say = initial(example_typepath.say_mod)
```

Reply:
Even with initial it runtimes for me that it cannot find the say_mod.